examples/docs_snippets/docs_snippets/integrations/tableau/customize-tableau-asset-defs.py

```diff
@@ -21,8 +22,20 @@
 # Tableau assets, such as the owners or asset key
 class MyCustomTableauTranslator(DagsterTableauTranslator):
     def get_sheet_spec(self, data: TableauContentData) -> dg.AssetSpec:
-        # We add a custom team owner tag to all sheets
-        return super().get_sheet_spec(data)._replace(owners=["team:my_team"])
+        # We create the default asset spec for a given sheet using super()
+        default_spec = super().get_sheet_spec(data)
+        # We customize the team owner tag only for sheets
+        return replace_attributes(default_spec, owners=["team:my_team"])
+
+    def get_asset_spec(self, data: TableauContentData) -> dg.AssetSpec:
```

Comment:
Does it make sense for a user to want to override both `get_asset_spec` and `get_sheet_spec`? I find this a little confusing since I would imagine the typical case would be to either override one or more of the "object-level" spec methods, or just `get_asset_spec`?

Maybe we ought to just consolidate on overriding `get_asset_spec`?

Reply:
That's fair - will update

Reply:
Done in [5eeb24c](https://github.com/dagster-io/dagster/pull/26029/commits/5eeb24cd31691e8511a6486240bfc66bcaf1d498)